config/config.js

```diff
@@ -39,15 +39,8 @@ nconf
   .file({file: path.join(__dirname, 'config.json')})
   // 4. Defaults
   .defaults({
-    // Typically you will create a bucket with the same name as your project ID.
-    CLOUD_BUCKET: '',
     DATA_BACKEND: 'datastore',
```

Comment:
We could remove it, it was in the original sample, we just need to remove it in few places where it is used to build model-datastore lib name.